clusterloader2/cmd/clusterloader.go

```diff
@@ -51,6 +51,7 @@ const (
 var (
 	clusterLoaderConfig config.ClusterLoaderConfig
 	testConfigPaths     []string
+	testSuiteConfigPath string
```

Comment:
Add a todo to remove testconfigpaths after we finish migration.

Same for flags.

Reply:
Done.